ui/src/adapters/api/credentials.ts

```diff
@@ -12,6 +12,7 @@ import {
 import { schemaParser } from "src/adapters/api/schemas";
 import { getStrictParser } from "src/adapters/parsers";
 import { Credential, Env, Schema } from "src/domain";
+import { Link, LinkAttributes, LinkStatus } from "src/domain/credential";
 import { API_VERSION, QUERY_SEARCH_PARAM, STATUS_SEARCH_PARAM } from "src/utils/constants";
 
 // TODO - refactor & order as Credentials are implemented
```

Comment:
How far along are we with this?

Reply:
We are still supporting old code

Reply:
@amonsosanz will handle `OldCredential` and this `TODO` in https://github.com/0xPolygonID/sh-id-platform/pull/297.